src/main/java/com/dev/cinema/service/AuthenticationService.java

```diff
@@ -0,0 +1,11 @@
+package com.dev.cinema.service;
+
+import com.dev.cinema.model.User;
+
+import javax.security.sasl.AuthenticationException;
+
+public interface AuthenticationService {
+    User login(String email, String password) throws AuthenticationException;
+
+    User register(String email, String password) throws AuthenticationException;
```

Comment:
```suggestion
    User register(String email, String password);
```
тут цього ексепшну не має бути

Reply:
fixed